SonarQube.MSBuild.Tasks/MergeRulesets.cs

```diff
@@ -128,24 +128,26 @@ private string TryResolveIncludedRuleset(string includePath)
 
         private void EnsureIncludeExists(XDocument ruleset, string includePath)
         {
-            if (IncludeExists(ruleset, includePath))
+            XElement includeElement = FindExistingInclude(ruleset, includePath);
```

Comment:
Currently we shouldn't actually be calling *MergeRulesets* with a primary/source ruleset that has an include in it, so we shouldn't really need to update existing includes. However, it was easier to update the existing code and test that to remove it.